docs/src/pages/components/lists/lists.md

```diff
@@ -47,7 +47,8 @@ Below is an interactive demo that lets you explore the visual results of the dif
 
 ## Align list items
 
-You should change the list item alignment when displaying 3 lines or more, set the `alignItems` prop to "flex-start".
+The vertical alignment of the avatar is wrong when displaying three lines or more.
```

Comment:
how does "wrong" look? "wrong" is an ambiguous term and oftentimes subjective. We should avoid "good" or "bad" or "right" or "wrong" in docs and just say explicitly what it refers to. For example
```suggestion
When displaying three lines or more, the avatar is not aligned at the top.
```